08-TabbedNavigation/src/PrismSample/Views/TabsPage.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<TabbedPage xmlns="http://xamarin.com/schemas/2014/forms"
```

Comment:
Be sure to update the Application Resources style for TabbedPage to include the following: `ApplyToDerivedTypes="True"`